### PR TITLE
BED-4753: Added styling selectors to address sections in api explorer not working in dark/light mode

### PR DESCRIPTION
**Pull request by specter-flq**
## Description

Added selectors to target elements that were not responding to the dark / light mode functionality in API explorer

## Motivation and Context

This addresses ticket [BED-4753](https://specterops.atlassian.net/browse/BED-4753)

## How Has This Been Tested?

Tested manually.

## Screenshots (optional):

https://github.com/user-attachments/assets/783c8e58-2c87-43cb-bc72-69af2900af47

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have met the contributing prerequisites
  - Assigned myself to this PR
  - Added the appropriate labels
  - Associated an issue: https://github.com/SpecterOps/BloodHound/issues/672
  - Read the Contributing guide: https://github.com/SpecterOps/BloodHound/wiki/Contributing
- [x] I have ensured that related documentation is up-to-date
  - Open API docs
  - Code comments (GoDocs / JSDocs)
- [x] I have followed proper test practices
  - Added/updated tests to cover my changes
  - All new and existing tests passed

[BED-4753]: https://specterops.atlassian.net/browse/BED-4753?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cmd/ui/src/App.tsx [hunk 1]
```diff
@@ -86,19 +88,39 @@ export const Inner: React.FC = () => {
                             outline: `1px solid ${theme.palette.color.links}`,
                         },
                     },
+                    '& .models': {
```
Comment (urangel) [inline cmd/ui/src/App.tsx:91]: Above '& .models', would you please add this styling for updating the "required" red font color 🙏 
```
                    '& .parameter__name.required::after': {
                        color: theme.palette.color.error,
                    },
```

**Review by urangel — APPROVED** [review]
Thank you for going through all of this. 🙇  It can be difficult to manage these views from external sources but it looks like you were meticulous about finding the styles that need updating!

File: cmd/ui/src/App.tsx (hunk 1 above)
Comment (specter-flq) [reply cmd/ui/src/App.tsx:93]: good catch thanks man. updated.

File: cmd/ui/src/App.tsx [hunk 2]
```diff
@@ -72,11 +72,13 @@ export const Inner: React.FC = () => {
                         & .response-col_links,
                         & .opblock-description-wrapper > p,
                         & .btn-group > button,
+                        & textarea,
+                        & select,
```
[the remark this answers is not in the document]
Comment (specter-flq) [reply cmd/ui/src/App.tsx:76]: good catch thanks again. updated.

**Pull request merged**